File: 자료구조 &알고리즘/Queue.py
class Queue():
    queue = []
    
    def enQueue(this, data):
        this.queue.append(data)
    
    def deQueue(this):
        return this.queue.pop()
    pass

queue = Queue()
queue.enQueue("첫 번째 데이터")
queue.enQueue("두 번째 데이터")
queue.enQueue("세 번째 데이터")

print(queue.deQueue())
print(queue.deQueue())
print(queue.deQueue())

File: 자료구조 &알고리즘/queue.java
import java.util.LinkedList;
// import java.util.Queue;

public class queue {
    static class Queue {
        private LinkedList<Integer> queue = new LinkedList<>();
        void enQueue(int n) {
            queue.add(n);
        }
        int deQueue () {
            return queue.poll();
        }
    };

    public static void main(String[] args) {
        Queue queue = new Queue();
        queue.enQueue(10);
        queue.enQueue(7);
        queue.enQueue(8);
        
        System.out.println(queue.deQueue());
        System.out.println(queue.deQueue());
        System.out.println(queue.deQueue());
    }

    public static void enQueue(int i) {
    }
}